migrations/sqls/20241014133216-alter-crm-v2-documents-defaults.js-down.sql

```diff
@@ -0,0 +1,5 @@
+-- Drop unique index on the licence number
+drop index crm_v2.documents_only_one_document_ref;
+
+-- Create unique index on licence number
+create unique index documents_document_ref on crm_v2.documents(regime, document_type, document_ref);
```

Comment:
Just to be on the safe side, I'd wrap both calls in a transaction.

```suggestion
BEGIN;

-- Drop unique index on the licence number
DROP INDEX crm_v2.documents_only_one_document_ref;

-- Create unique index on licence number
CREATE UNIQUE INDEX documents_document_ref ON crm_v2.documents(regime, document_type, document_ref);

COMMIT;
```